mlir-sycl/include/mlir/Dialect/SYCL/IR/SYCLOpsTypes.h

```diff
@@ -53,6 +53,8 @@ enum class AccessAddrSpace : int {
   ExtIntelHost = 6
 };
 
+enum class DecoratedAddress : int { No = 0, Yes = 1, Legacy = 2 };
```

Comment:
I'd say `AccessDecorated` is a better name for this.

Reply:
How about `DecoratedAccess`? :)